### PR TITLE
Stats - Followers module - Duplicate pagination item to the top of screen

### LINKED ISSUE #2201: Duplicate the pagination at the top of the screen as well
Due to the length of the list in the Followers detail view, and the need to have pagination, I think it would be best to duplicate the pagination at the top of the screen as well. Perhaps this could only be done on devices where all items can’t be shown without scrolling, that would be a nice touch.

### PR DESCRIPTION
**Pull request by daniloercoli**
Fix #2201 by adding a new UI pagination widget at the top of the screen.
Also removed unused file.

/cc @jancavan

### CODE REVIEW

**Comment by jancavan** [comment]
@daniloercoli Looks good. One more small change - would it be possible to make the text size a bit smaller? I guess it's unavoidable for some words to be moved to the second line, but it'll give us more room. Thanks.

pagination2

**Comment by daniloercoli** [comment]
@jancavan - Text size is not set to size-small.

**Comment by jancavan** [comment]
@daniloercoli Looks good :+1:

File: WordPress/src/main/java/org/wordpress/android/ui/stats/StatsFollowersFragment.java [hunk 1]
```diff
@@ -125,69 +125,106 @@ protected void updateUI() {
...
-                    mPaginationContainer.setVisibility(View.VISIBLE);
-                    mPaginationText.setText(
-                            String.format(
-                                    getString(R.string.stats_pagination_label),
-                                    followersModel.getPage(),
-                                    followersModel.getPages()
-                            )
+                    mBottomPaginationContainer.setVisibility(View.VISIBLE);
+                    mTopPaginationContainer.setVisibility(View.VISIBLE);
+                    String paginationLabel = String.format(
```
Comment (nbradbury) [inline WordPress/src/main/java/org/wordpress/android/ui/stats/StatsFollowersFragment.java]: Based on the screenshot below, thousands-separators would be good here.

screen shot 2015-02-12 at 5 52 19 pm

**Comment by nbradbury** [comment]
Looks good, just waiting for travis so I can merge :shipit:

**Pull request merged**